src/components/Header/Header.js

```diff
@@ -1,9 +1,12 @@
 import './Header.css'
 
-const Header = () => {
+const Header = ({ user }) => {
```

Comment:
It might make sense to pass in user.name instead of the whole user object as props, since that's the only piece of info that we are using in the header.